weltyt.py

```diff
@@ -167,6 +171,14 @@ def buy_lands(lands):
                 \
                 '
             )
+        
+        elif what_buy == "Help":
```

Comment:
Move all possible choices (`help`, `plants` etc.) to some kind of tuple or dict.